3rdParty/rocksdb/6.2/CMakeLists.txt

```diff
@@ -471,6 +471,8 @@ include_directories(${PROJECT_SOURCE_DIR}/include)
 include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/third-party/gtest-1.7.0/fused-src)
 find_package(Threads REQUIRED)
 
+add_subdirectory(third-party/gtest-1.7.0/fused-src/gtest)
```

Comment:
could we rather do this one layer up?

Reply:
I would like to keep it in RocksDB's CMake file.
It is originally in there in upstream RocksDB, and I would like to keep the changes minimal.

Reply:
We may eventually bundle a different version of gtest inside 3rdParty directly, and use that instead of the version that comes with RocksDB.
We don't need to do this now, but long-term it may be a good idea to decouple things.

Reply:
but since we want to change stuff as little as possible, can't we add it in 3rdParty/rocksdb/CMakeList.txt ? since we don't want to build the rocksdb unittests?
then you wouldn't need a change in rocksdb's cmake file.

Reply:
We have two other adjustments in RocksDB's cmake file anyway:
- use c++14 instead of c++11
- don't regenerate the build timestamp on every build, but cache it.

Reply:
So unless we are not suggesting these adjustments to the upstream RocksDB library, we will have to do these small modifications still.
And as I said, we can add another PR at any time it is convenient that makes ArangoDB use its own version of gtest, so relying on the RocksDB-bundled version of gtest can be avoided.

Reply:
I would, as @dothebart suggested it, put the `add_subdirectory()` call in the top-level (3dParty) CMakeLists.txt. But technically this is equally good, as the coupling to the RocksDB version would only feel less while the relevant code is distributed over more files. As we might eventually switch to a standalone version anyways I have no strong opinion.